modules/nf-core/svync/tests/main.nf.test

```diff
@@ -46,8 +46,29 @@ nextflow_process {
             assertAll(
                 { assert process.success },
                 { assert snapshot(
-                    process.out.vcf[0][0],
-                    path(process.out.vcf[0][1]).linesGzip[20..30]
+                    process.out.collectEntries { key, value ->
+                        if(key.matches("^\\d+\$")) {
+                            return null
+                        }
+                        else if(key == "vcf") {
+                            return [
+                                key,
+                                value.collect { meta, vcf ->
+                                    [ meta, "${file(vcf).name},variantsMD5:${path(vcf).vcf.variantsMD5}" ]
+                                }
```

Comment:
Is it necessary to have MD5 checksum in the pair?

Reply:
What do you mean by that? :face_with_spiral_eyes:

Reply:
Wouldn't this be enough: `meta, "${file(vcf).name}"`?
Or I might miss something here with collect :)

Reply:
Aah no this outputs the file name and the MD5sum of the variants in the VCF since the header is unstable. You'll get an output like this: `test.vcf.gz,variantsMD5:<md5sum>`

It just looks nice IMO :grin: